cmd/tidb-dashboard/main.go

```diff
@@ -69,6 +69,7 @@ func NewCLIConfig() *DashboardCLIConfig {
 	flag.BoolVar(&cfg.CoreConfig.EnableExperimental, "experimental", cfg.CoreConfig.EnableExperimental, "allow experimental features")
 	flag.StringVar(&cfg.CoreConfig.FeatureVersion, "feature-version", cfg.CoreConfig.FeatureVersion, "target TiDB version for standalone mode")
 	flag.IntVar(&cfg.CoreConfig.NgmTimeout, "ngm-timeout", cfg.CoreConfig.NgmTimeout, "timeout secs for accessing the ngm API")
+	flag.BoolVar(&cfg.CoreConfig.EnableKeyVisualizer, "keyVisualizer", true, "enable/disable key visualizer(default: true)")
```

Comment:
`keyVisualizer` is better to use `keyviz` as flag name to keep consistence with other flags.

Reply:
@baurine,
https://github.com/pingcap/tidb-operator/pull/5536
tidb-operator PR that uses keyVisualizer name as command line arg already merged :(
changing it to keyviz would require another PR to tidb-operator

let me know if you still want me to rename it or it is ok to keep keyVisualizer

Reply:
It seems tidb-operator use the camel naming style, so `keyVisualizer` is fine, while tidb-dashboard use the `xx-yy` naming style, I think it's better to keep the same in one project.

Reply:
@baurine,
renamed to keyviz
please re-review